cluster/addons/podsecuritypolicies-rbac/gce/kube-system-binding.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: RoleBinding
+metadata:
+  name: system:podsecuritypolicy:unprivileged-addon
+  namespace: kube-system
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+    kubernetes.io/cluster-service: "true"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: system:podsecuritypolicy:unprivileged-addon
+subjects:
+- kind: Group
+  # All service accounts in the kube-system namespace are allowed to use this.
+  name: system:authenticated
```

Comment:
this is more broad than the comment.. intentional?

Reply:
Ah, I didn't know about the serviceaccounts group. Done.

I think this is effectively equivalent though, unless it's possible to run a pod without a service account?